pixi.toml

```diff
@@ -155,6 +155,7 @@ polars = "*"
 reacton = "*"
 scipy = "*"
 textual = "*"
+pydantic = "*"
```

Comment:
It's best to add a minimum pin, as the code is likely not to be compatible with all the versions of pydantic. And yes, most of the dependencies there have no pins, it doesn't mean that's the right way :)